### PR TITLE
[TRUS-2673] - Added UTR Page

### PR DESCRIPTION
**Pull request by jassalrichy**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -20,3 +20,8 @@ GET        /status/closed                               controllers.EstateStatus
 GET        /status/problem-with-service                 controllers.EstateStatusController.problemWithService()
 GET        /status/utr-does-not-match-records           controllers.EstateStatusController.utrDoesNotMatchRecords()
 GET        /status/account-not-linked                   controllers.EstateStatusController.accountNotLinked()
+
+GET        /UTR                                         controllers.UTRController.onPageLoad(mode: Mode = NormalMode)
```
Comment (adamconder) [inline conf/app.routes:24]: can this be /unique-tax-reference-number

File: app/pages/UTRPage.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object UTRPage extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "uTR"
```
Comment (adamconder) [inline app/pages/UTRPage.scala:25]: Can this be all lowercase

File: app/utils/CheckYourAnswersHelper.scala [hunk 3]
```diff
@@ -28,6 +28,15 @@ import CheckYourAnswersHelper._
 
 class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) {
 
+  def uTR: Option[AnswerRow] = userAnswers.get(UTRPage) map {
+    x =>
+      AnswerRow(
+        HtmlFormat.escape(messages("uTR.checkYourAnswersLabel")),
```
Comment (adamconder) [inline app/utils/CheckYourAnswersHelper.scala:34]: Does this correspond to a message key?

File: conf/app.routes (hunk 1 above)
Comment (jassalrichy) [reply conf/app.routes:24]: Updated

File: app/pages/UTRPage.scala (hunk 2 above)
Comment (jassalrichy) [reply app/pages/UTRPage.scala:25]: Updated

File: app/utils/CheckYourAnswersHelper.scala (hunk 3 above)
Comment (jassalrichy) [reply app/utils/CheckYourAnswersHelper.scala:34]: Updated

**Pull request merged**